### DISCUSSION TITLE
Huggingface model not working

### DISCUSSION THREAD

**Discussion by lucy0828** [explosion/spacy-llm#438] [Q&A]
I am trying to use OpenLLaMA for TextCat. I have already tested my code with OpenAI API and it is classifying the text very well. 
However, if I run it with any models from Huggingface, the output of labels are always 0. 

![CleanShot 2024-02-04 at 13 02 10](https://github.com/explosion/spacy-llm/assets/46836844/fa83dce9-a6c5-490e-8736-220d3c54ed19)

![CleanShot 2024-02-04 at 13 02 26](https://github.com/explosion/spacy-llm/assets/46836844/aa76ff84-b78e-4ad9-96e4-62e7eda5635e)

![CleanShot 2024-02-04 at 13 02 39](https://github.com/explosion/spacy-llm/assets/46836844/d378e68f-80ce-4917-80e5-0311c3c60e67)

What might cause this issue?

**Comment by lucy0828** [comment]
![CleanShot 2024-02-04 at 13 21 12@2x](https://github.com/explosion/spacy-llm/assets/46836844/d8b83de3-77cd-494d-b8e2-bcad627cdf93)
This is my output

**Comment by rmitsch** [comment]
I recommend setting `save_io = True` in your config (in `components.llm`) and investigating your raw model output. OpenLLaMA 3b is a comparatively old and small model, so it's quite possible that it returns wrong/non-parseable outputs.
